NEWS.md

```diff
@@ -1,5 +1,9 @@
 # usethis (development version)
 
+* `use_news_md()` now places "(development version)" in the header of `NEWS.md`
```

Comment:
Does this need a bit more qualification / rewording? `use_news_md()` did already place "(development version)" in most relevant scenarios, I think, so the news is that we're making it more robust.

Reply:
Oh wait, I am thinking of subsequent edits to `NEWS.md`. Got it. Disregard.

Reply:
Yep, I had to keep remembering that too!